ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -932,11 +953,17 @@ open class TableViewCell: UITableViewCell {
     }
 
     @objc public override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        let tokens = TableViewCellTokens()
+        self.tokens = tokens
+
         super.init(style: .default, reuseIdentifier: reuseIdentifier)
         initialize()
     }
 
     public required init?(coder aDecoder: NSCoder) {
+        let tokens = TableViewCellTokens()
```

Comment:
shouldn't this be the preconditionFailure?

`preconditionFailure("init(coder:) has not been implemented")`

Reply:
Looks like this control has historically supported `NSCoder` inits, necessary for nib/storyboard initialization. I'd rather not remove that support if we've already shipped it, since it's unclear who might be relying on it.

Reply:
Why do some controls have the preconditionFailure, and others continue with the initialization? @mischreiber @rdeassis  (am a little unfamiliar/confused as I looked through the other controls!)

Reply:
Honestly, I assume the answer is historical. We haven't prioritized supporting `NSCoder` initialization for new controls, but presumably some older controls did in fact support that.

I don't have a strong preference on what we _should_ do here, but I definitely see the value in consistency one way or another.